controllers/managedocs_controller.go

```diff
@@ -1110,6 +1119,8 @@ func (r *ManagedOCSReconciler) reconcileK8SMetricsServiceMonitorAuthSecret() err
 		var err error = nil
 		if auth, err = r.readGrafanaV1Secret(); err != nil {
 			if errors.IsNotFound(err) {
+				// Log the error irrespective of readGrafanaV2Secret result
+				r.Log.Error(err, "Unable to find grafana-datasources secret")
```

Comment:
This is not an error by definition. It is ok to log but lets keep that as an `info`